skuber/src/main/scala/com/virtuslab/iat/skuber/SimpleDeployer.scala

```diff
@@ -107,38 +107,63 @@ object SimpleDeployer extends SimpleDeployer {
       executor: ExecutionContext,
       client: K8SRequestContext,
       lc: LoggingContext
-    ): Either[Throwable, A] =
-    Try {
-      val future = futureUpsert(o)
-      val result = Await.result(future, 1.minute)
-      println(s"Successfully created or updated '$result' on Kubernetes cluster")
-      result
-    }.toEither
+    ): Future[A] = {
+    futureUpsert(o)
+  }
+
+  def upsertBlocking[A <: ObjectResource: Format: ResourceDefinition](
```

Comment:
For another PR: maybe make the timeout (currently just `1.minute`) an optional param